README.md

```diff
@@ -32,6 +32,13 @@ issue](https://github.com/jannis-baum/vivify/issues/new/choose) or
 - links to other files: [relative links like in
   GitHub](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links)
   as well as absolute file links
+- [add styles, classes, ids or other attributes directly from
+  Markdown](https://github.com/arve0/markdown-it-attrs?tab=readme-ov-file#examples)
+
+You can find examples for all supported features in the files in the
+[`tests/`](tests) directory. In case you are looking at these on GitHub, keep in
+mind that GitHub doesn't support some of the features that Vivify supports so
+some things may look off.
```

Comment:
Yeah I quite like this clarification because one could easily browse this on github file preview and think it's broken when it isn't for us :D